Makefile

```diff
@@ -55,3 +55,33 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+DURATION ?= 3s
+COUNT ?= 5
+BENCH_PATH ?= bench
+TEST_PATH ?= ./...
+
+.PHONY: bench-deps
+bench-deps:
```

Comment:
I see inconsistency in new target names: two names uses `bench` prefix and two names uses `perf`. I would set a single one to avoid misleading.

Reply:
Fixed.